src/families/cardano/serialization.ts

```diff
@@ -0,0 +1,80 @@
+import { BigNumber } from "bignumber.js";
+import type {
+  CardanoOutput,
+  CardanoOutputRaw,
+  CardanoResources,
+  CardanoResourcesRaw,
+  PaymentCredential,
+  PaymentCredentialRaw,
+  StakeCredential,
+  StakeCredentialRaw,
+  Token,
+  TokenRaw,
+} from "./types";
+
+function toTokenRaw(r: Token): TokenRaw {
+  return { ...r, amount: r.amount.toString() };
+}
+
+function fromTokenRaw(r: TokenRaw): Token {
+  return { ...r, amount: new BigNumber(r.amount) };
+}
+
+function toPaymentCredentialRaw(r: PaymentCredential): PaymentCredentialRaw {
+  return r;
+}
+
+function fromPaymentCredentialRaw(r: PaymentCredentialRaw): PaymentCredential {
+  return r;
+}
+
+function toStakeCredentialRaw(r: StakeCredential): StakeCredentialRaw {
+  return r;
+}
+
+function fromStakeCredentialRaw(r: StakeCredentialRaw): StakeCredential {
+  return r;
+}
```

Comment:
I think these work because by chance `BipPath` and `BipPathRaw` are identical, but it would be safer to do the explicit conversion here.